inference-engine/ie_bridges/python/cmake/UseCython.cmake

```diff
@@ -288,7 +288,7 @@ function( cython_add_module _name )
   compile_pyx( ${_name} generated_file ${pyx_module_sources} )
   python_add_module ( ${_name} ${generated_file} ${other_module_sources} )
   target_include_directories( ${_name} PRIVATE ${PYTHON_INCLUDE_DIRS})
-  # set_target_properties(${_name} PROPERTIES PREFIX "" SUFFIX "${PYTHON_MODULE_EXTENSION}")
+  set_target_properties(${_name} PROPERTIES PREFIX "" SUFFIX "${PYTHON_MODULE_EXTENSION}")
```

Comment:
I suppose it does not work in we build python in standalone mode w/o the whole openvino (pybind11 which provides `PYTHON_MODULE_EXTENSION` variable)